### ISSUE TITLE
Custom Domain Support

### ISSUE THREAD

**Issue by gazpachoking** [conceptualspace/shuffle-for-plex#1]
This is great, been looking for something like this. Would be nice if an option could be added so this works when I access my plex server at a custom domain, rather than at plex.tv.

**Comment by conceptualspace** [comment]
good idea. i wasn't sure how to implement it without requesting access to all urls, but maybe it can be set as an optional permission. will try it out

 https://developer.chrome.com/apps/permissions

**Comment by gazpachoking** [comment]
Not up on my chrome extension dev skills, but that certainly looks like a promising lead.
> If you want to request hosts that you only discover at runtime, include `"http://*/"` and/or `"https://*/"` as an `optional_permission`.
